user-guides/data-farming/README.md

```diff
@@ -0,0 +1,49 @@
+---
+description: Learn the ins-and-outs Data Farming
+---
+
+# Data Farming Guides
+
+<figure><img src="../../.gitbook/assets/gif/farming.gif" alt=""><figcaption></figcaption></figure>
+
+Data Farming is the core incentive system inside of Ocean Protocol. It offers different ways for participants to help the protocol grow and succeed.
+
+It's broken down into 2 reward streams:
+1. Passive Rewards
+2. Active Rewards
+
+As the name suggests, users need to do very little to earn Passive Rewards and the program is aimed to onboard as many people as possible to help drive liquidity and conviction into the protocol.
+
+Active Rewards is an evolving incentive program that offers different venues for contributors to complete active work that drives the protocol forward.
+
+Anyone is welcomed to participate.
+
+## Passive Rewards stream
```

Comment:
It seems like we're being a bit redundant. The user guides are meant to offer straightforward how-to's for non-technical users. To avoid repetition, let's consider adding links to detailed information in the rewards section. This way, users can access more in-depth explanations if needed. Simplify and streamline – that's the way to go! 📚💡

Reply:
Please seem the READMEs for Sell-NFT-Computations and Host-Assets.

They both provide a high-level overview of each section inside the user-guides and then links to all the appropriate documents.

I have already moved most of this redundant information to the right docs inside of `/rewards/`.

It now simply uses the same outline as those 2 base READMEs.

Resolving this.

Reply:
https://github.com/oceanprotocol/docs/issues/1234